downward_force_controller:
  type: softgrasp_ros_controllers/DownwardForceController
  arm_id: panda
  joint_names:
    - panda_joint1
    - panda_joint2
    - panda_joint3
    - panda_joint4
    - panda_joint5
    - panda_joint6
    - panda_joint7

tf_policy_controller:
  type: softgrasp_ros_controllers/TFPolicyController
  arm_id: $(arg arm_id)
  decision_step: 30
  success_thresh_x: 0.1
  success_thresh_y: 0.15
  success_thresh_z: -0.02
  policy_action_name: $(arg policy_name)_policy
  joint_names:
    - $(arg arm_id)_joint1
    - $(arg arm_id)_joint2
    - $(arg arm_id)_joint3
    - $(arg arm_id)_joint4
    - $(arg arm_id)_joint5
    - $(arg arm_id)_joint6
    - $(arg arm_id)_joint7  